book/tutorials/sar/swesarr.ipynb

```diff
@@ -69,12 +68,32 @@
    "id": "worthy-dimension",
    "metadata": {},
    "source": [
-    "## What is SWESARR?\n",
+    "## What is SWESARR?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5cb68a4f-3e5b-4d29-882f-b02cdc524392",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Audio,Image, YouTubeVideo;    id='5hVQusosGSg';    YouTubeVideo(id=id,width=600,height=300,start=210,end=238)\n",
```

Comment:
nice! didn't know you could do this. i'm not 100% sure the video will show up on the website, but definitely works when running the notebook interactively.

Reply:
I thought it was really neat! I would prefer to just code it in HTML, but I couldn't quite get the **stop** flag to work for me.